benchmarks/spack/isambard3/grace/spack.yaml

```diff
@@ -25,6 +23,18 @@ spack:
       environment: {}
       extra_rpaths: []
   packages:
+    cmake:
+      externals:
+      - spec: cmake@3.20.4
+        prefix: /usr
+    python:
+      externals:
+      - spec: python@3.11.7+bz2+crypt+ctypes+dbm+lzma+pyexpat+pythoncmd+readline+sqlite3+ssl+tix+tkinter+uuid+zlib
+        prefix: /lfs1i3/home/lexci-i3/tkoskela.lexci-i3/python_envs/test
```

Comment:
This is in your home
```suggestion
```

Reply:
Oops. I see, it had picked up the python from my virtual environment. Well spotted!